### ISSUE TITLE
Classification_3d densenet_training_array example bug

### ISSUE THREAD

**Issue by MarcosVillacanas** [Project-MONAI/MONAI#546]
**Describe the bug**
When trying to run Classification_3d densenet_training_array example, Run-Time Exception is thrown because of data type conflict. Torch target attribute, which corresponds to Tensor, is expected to be of type Long, but it is Int.

**To Reproduce**
1. Get the data used in this example from https://brain-development.org/ixi-dataset/
2. Save it in the root of the project with this structure workspace\data\medical\ixi\IXI-T1\xx.gz
3. Run the example

**Expected behavior**
There should be five epochs training process, but the execution is stopped due to this Run-Time Exception.

**Screenshots**
![screenshot](https://user-images.githubusercontent.com/48330663/84478413-8d5ae080-ac91-11ea-8c2d-02b8c5df21ed.png)

**Environment**
 - Windows 10
 - Python version 3.7.5
 - MONAI version 0.1.0 Commit 01e4457d72acd228b5da0e91dcc0778adaf6b101 (06/11/2020)
 - Pytorch version: 1.5.0+cu101
 - GPU NVIDIA GeForce MX150

**Possible Fix**
This error can be fixed by altering functional.py inside torch package, changing:
![imagen](https://user-images.githubusercontent.com/48330663/84478589-e296f200-ac91-11ea-83b2-0332b0da98de.png)
to:
![imagen](https://user-images.githubusercontent.com/48330663/84478712-08bc9200-ac92-11ea-8aa7-c9bbab574b9f.png)

So that the Int is cast to Long.

**Comment by Nic-Ma** [comment]
Hi @MarcosVillacanas ,

Thanks for your bug report.
I think this issue may be caused by different default data type on windows and ubuntu.
So I committed a PR https://github.com/Project-MONAI/MONAI/pull/547 to fix the label data type to `np.int64`.
Could you please help checkout the repo and branch of above PR to verify again?

Thanks.

**Comment by MarcosVillacanas** [comment]
I have already checked your PR changes, works fine! 
The bug is fixed :)

**Comment by Nic-Ma** [comment]
Cool!
